internal/osutils/autostart/autostart_darwin.go

```diff
@@ -20,6 +20,11 @@ const (
 )
 
 func enable(exec string, opts Options) error {
+	err := cleanLegacyFile(exec, opts)
+	if err != nil {
+		return errs.Wrap(err, "Could not clean legacy file")
+	}
```

Comment:
We just started moving this logic to the installer, might be worth waiting for David's PR and piggy backing off his changes:

https://github.com/ActiveState/cli/pull/2456/files/f7027b520621686fcac30ec61731c2c03c58955b..479edd891a2492b2f2cd7d9af55e1b8839a96e92